_posts/en/newsletters/2020-11-11-newsletter.md

```diff
@@ -0,0 +1,139 @@
+      If we can't use 64 bit limbs, then 32 bit limbs are used instead."
+  a2link="https://bitcoincore.reviews/19055-2#l-190"
+
+  q3="What is 1437 mod 99?"
+  a3="1437 mod 99 = 51. This question was given as an example of an efficient way to
+      calculate the remainder of a number when divided by a modulus that is slightly
+      smaller than a power of the number base. In this example, 99 is slightly
+      smaller than 10^2, so to calculate the remainder we can add the bottom half
+      (37 in this case) to the top half times the difference of the modulus from the
+      base number power (14 * (10^2 - 99) in this case). The same technique is used
+      in the MuHash implementation and explains why the modulus was chosen as a
+      number slightly smaller than 2^3072."
+  a3link="https://bitcoincore.reviews/19055-2#l-246"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:none as of Sunday; harding to update or delete this section Tuesday
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [BOLTs #807][] Fail channel in case of high-S remote signature reception FIXME:dongcarl
+
+- [Eclair #1545][] adds a blockchain watchdog to detect eclipse attacks. Eclair
+  fetches block headers from multiple third-party sources to countercheck the
+  best chain provided by its peers. As described in [Newsletter #101][news101 time-dilation],
+  an attacker could use an eclipse attack to hide a malicious channel closing
+  attempt until the dispute period expires.
+
+- [LND #4735][] adds as `max_local_csv` configuration parameter that
+  will reject channels from peers who require the local node wait more
+  than the indicated number of blocks before being able to spend its
+  funds if it unilaterally closes the channel.  The default maximum is
+  1,000 blocks, and there's also a minimum of 144 blocks to ensure users
+  don't set the value lower than is reasonable.
+
+- [LND #4701][] enables the `assumechanvalid` (assume channels are
+  valid) configuration option by default.  For nodes using the Neutrino
+  client to retrieve [compact block filters][topic compact block
+  filters], this will assume the channels the node learns about via LN
+  gossip are available.  If they're not actually available (e.g. they've
+  been closed recently), any payments LND attempts to route through
+  those channels will fail.  This may cause delays or unnecessary
+  payment failures but not the loss of funds.
```

Comment:
Perhaps mention the tradeoffs a bit, e.g. from this paragraph in the pull, the option avoids long initial verification rescans on startup, but at the risk of an incorrect view of the network.

```
; Skip checking channel spentness and existence during graph validation for
; neutrino. Enabling this option means that neutrino nodes will not need to
; perform long rescans which block initial usage of the daemon, but comes at
; the cost of not validating channels in your routing graph. Skipping this
; validation means that your node may have an incorrect view of the network
; if it receives updates for closed or non-existent channels. This could affect
; routing, but funds are safu.
```

Reply:
￼Discussed with PR author Carla, who very helpfully mentioned, "Just a nit on assumechanvalid, it’s not enabled by default, we just removed the experimental build flag (people still need to opt in)." The reason for the PR: "people were confused when we told them about it but it wasn’t included in the binary when they built it."

Reply:
(1) Thanks for noticing that I focused on the downside but forgot to describe the upside; (2) HUGE thanks for verifying with Carla and allowing me to fix my error before we published.  I appreciate it!

Reply:
Thanks to Carla for having a look! 🌳